.github/workflows/macos.yml

```diff
@@ -1,41 +1,75 @@
 name: Celix MacOS
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
 
 jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [macOS-latest]
-        compiler: [clang]
+
+  build-conan:
+    if: false #See #471
+    runs-on: macOS-latest
+    timeout-minutes: 120
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3.3.0
+      - name: Install conan
+        run: |
+          brew install conan
+      - name: Setup Conan Profile
+        run: |
+          conan profile new default --detect
+          conan profile update settings.build_type=Debug default
+      - name: Install Dependencies
+        env:
+          CONAN_BUILD_OPTIONS: |
+            -o celix:enable_testing=True
+            -o celix:enable_address_sanitizer=True
+            -o celix:build_all=True
+        run: |
+          #force reequire libcurl 7.64.1, due to a sha256 verify issue in libcurl/7.87.0
+          conan install . celix/ci -pr:b default -pr:h default -if build ${CONAN_BUILD_OPTIONS} -b missing  --require-override=libcurl/7.64.1
+      - name: Build
+        run: |
+          conan build . -bf build --configure
+          conan build . -bf build --build
+      - name: Test
+        run: |
+          cd build
+          source activate_run.sh
+          ctest --verbose
+          source deactivate_run.sh
+
```

Comment:
We need another step running `conan create -tf examples/conan_test_package ...` to verify that every artifact is installed properly. 

Note that we have add several C++ options, we should try to include these C++ headers in test_package.